lib/improver/database.py

```diff
@@ -232,23 +232,26 @@ class VerificationTable(SpotDatabase):
 
     """
 
-    def __init__(self, output, outfile, primary_dim, tablename, experiment_ID):
-        super(VerificationTable, self).__init__(output, outfile, tablename, primary_dim)
+    def __init__(self, output, outfile, tablename, primary_dim, experiment_ID,
+                 max_forecast_lead_time):
+        super(VerificationTable, self).__init__(output, outfile,
+                                                tablename, primary_dim)
         self.primary_dim = 'time'
         self.primary_map = ['validity_date', 'validity_time']
         self.primary_func = [lambda x:dt.utcfromtimestamp(x).date(),
                              lambda x:dt.utcfromtimestamp(x).hour*100]
 
         self.pivot_dim = 'forecast_period'
         self.pivot_map = lambda x: 'fcr_tplus{:03d}'.format(int(x/3600))
-        self.pivot_max = 54*60*60
 
         self.column_dims = ['wmo_site', 'name']
         self.column_maps = ['station_id', 'cf_name']
+        self.coord_to_slice_over = "index"
 
         if experiment_ID:
-            self.column_dims = self.column_dims + ["exp_id"]
-            self.column_maps = self.column_maps + [experiment_ID]
+            self.column_dims = self.column_dims + [experiment_ID]
+            self.column_maps = self.column_maps + ["exp_id"]
+        self.max_forecast_lead_time = max_forecast_lead_time
 
     def ensure_all_pivot_columns(self, dataframe):
         """
```

Comment:
I think this function should either be, ensure_all_forecast_ranges (speicific) or ensure_all_pivot_columns (generic) and take the argument maximum_pivot_value or similar, otherwise currently it imply it's generic, but is currently not.

Reply:
Yep, happy to change the name to something more specific